### PR TITLE
Refactor docs

### PR DESCRIPTION
**Pull request by mkborregaard**
Richard, could you take a look at whether you are OK with me taking the docs this way? Maybe we could just merge all these tomorrow, then you can judge by looking at the docs -they this is of course in principle WIP.
I'd like to move forward with this in preparation of the course.

### CODE REVIEW

File: docs/src/man/routes.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+branchhistory
+branchfuture
+branchroute
+nodehistory
+nodefuture
+noderoute
+distance
+distances
+heighttoroot
+heightstoroot
```
Comment (richardreeve) [inline docs/src/man/routes.md:13]: These are definitely queries (above), but are they routes?
```suggestion
```

**Review by richardreeve — APPROVED** [review]
Looks good to me - I'm happy to have structure to the docs! I just wasn't sure about one issue with the routes section.

File: docs/src/man/routes.md (hunk 1 above)
Comment (mkborregaard) [reply docs/src/man/routes.md:13]: I guess not really, maybe we should move them - I have them here because they are defined in `routes.jl` though.

**Comment by mkborregaard** [comment]
I've kept all of your text except the bit where you say that the only thing this package really cares about is Diversity.jl

File: docs/src/man/routes.md (hunk 1 above)
Comment (richardreeve) [reply docs/src/man/routes.md:13]: Yes, I suggest in #63 that you move them from `routes.jl` to `metrics.jl` now that `metrics.jl` exists, so I think removing them from `routes.md` makes sense too.

**Comment by richardreeve** [comment]
Again, the errors seem to be from the grandparent, so apart from the `routes.md` issue above I think this is good to merge too.

**Comment by richardreeve** [comment]
@mkborregaard So, do you want to just delete those routes-that-aren't routes and merge this to reduce the complexity of the PR chain?

**Comment by mkborregaard** [comment]
Yeah I've actually done everything you asked but something seems to have gone wrong during the rebases. However the final version works, so let me just merge.

**Pull request merged**